waku/v2/node/start_network2.nim

```diff
@@ -153,12 +212,15 @@ when isMainModule:
 
   # Scenario xx2 14
   let amount = conf.amount
+  let numFleetNodes = conf.numFleetNodes
```

Comment:
If this is a new scenario I'd just copy paste existing code and add it there instead of overwriting

>  Generalize the setup to allow for various disovery methods, especially static nodes. In its current state, this PR simply overwrites the former experiment, using discv5 instead of static nodes.

Shouldn't need to generalize too much, just add another code section for discv5 scenario. Let's KISS

Reply:
Done :).